specificatie/openapi.yaml

```diff
@@ -159,6 +159,66 @@ paths:
         default:
           $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/default'
 
+  #19, 306, 373 
+  /adressen/_zoek:
+    post:
+      requestBody:
+        description: Zoek criteria waarmee naar adressen moet worden gezocht.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AdressenZoekObject'
+      security:
+        - apiKeyBAG: []
+      operationId: zoekAdressenInContour
+      tags:
+        - Adres
+      summary: "Vind adressen die binnen de opgegeven contour vallen, evt. op basis van bouwjaar."
+      description: "Zoek binnen een geometrische contour naar adressen, evt. op. basis van bouwjaar. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.1.0/features/fields.feature)."
+      parameters:
+        - $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/fields'
```

Comment:
we combineren nu parameters in de body en parameters in de uri. @MelvLee is dat gewenst/common Practice?

Reply:
besproken en de conclusie is dat dit prima om filters in body te stoppen en representatie (fields, expand, page) in query

Reply:
OK